### DISCUSSION TITLE
Passing external design tokens to Fast component library

### DISCUSSION THREAD

**Discussion by anuragxxd** [microsoft/fast#6675] [Q&A]
# 💬 RFC

Actually what we are trying to do is develop the web-component library with the help of the fast. But we want it to be headless in terms of design so that the user using our library would we just able to pass the external design tokens of their choice & the component as the whole can render according to there own tokens. If there any functionality in the fast that easies this process. I have gone through the docs but haven't found any solid foundation about this type to functionality. 

It would we great if you can see if this is possible & point me out to any docs I may have missed!

**Comment by chrisdholt** [comment]
Converting this to a discussion as it seems more like Q&A than an issue.

**Comment by anuragxxd** [comment]
Hey @chrisdholt, Any solution you have regarding this?
